src/getCompressionType.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const zlib = require('zlib');
+
+const BROTLI = 'brotli';
+const DEFLATE = 'deflate';
+const GZIP = 'gzip';
+
+const getCompressionType = (compressionType) => {
+  switch (compressionType) {
+    case BROTLI:
+      return zlib.createBrotliCompress();
+
+    case DEFLATE:
+      return zlib.createDeflate();
+
+    case GZIP:
+    default:
```

Comment:
The 'default' case in a switch statement should be separate, not be combined with another case 'GZIP'. If 'GZIP' needs to be the default, consider removing the 'default' keyword and arranging the conditions so that GZIP will be selected if no other conditions are met.